apis/pad-systems/active-pad-system/api/active-pad-system.yaml

```diff
@@ -61,17 +61,23 @@ components:
       type: object
       required:
         - StationID
+        - MobilePlatform
         - BiometricSample
         - PADAnalysis
       properties:
         StationID:
           type: string
           description: ID of the station who is submitting this image. IDs will be provided to vendors upon installation for the test event.
           example: Station_A
+        MobilePlatform:
+          type: string
+          enum:
+            - iOS
+            - Android
         BiometricSample:
           type: string
           description: The captured biometric sample, encoded as a base64 string.  This can be an image, encoded as a PNG or JPEG
-                       or a short (<15s) video, encoded as a MOV.
+                       or a short (<15s) video, encoded as a MOV or a MP4.
```

Comment:
Should change <15s to <10s

Reply:
Additionally, we should add here or in the endpoint description that a maximum size of the request is 100Mb.